examples/complex/jsondump.py

```diff
@@ -0,0 +1,257 @@
+"""
+This script serializes the entire traffic dump, including websocket traffic,
+as JSON, and either sends it to a URL or writes to a file. The serialization
+format is optimized for Elasticsearch; the script can be used to send all
+captured traffic to Elasticsearch directly.
+
+Usage:
+
+    mitmproxy
+        -R http://example.com/
```

Comment:
To use mitmproxy in reverse mode please use `--mode reverse:SPEC` instead:
https://docs.mitmproxy.org/stable/concepts-modes/#reverse-proxy

Reply:
Fixed, thanks!